host/src/gap.rs

```diff
@@ -92,31 +92,4 @@ impl<'a> GapConfig<'a> {
             appearance: &appearance::GENERIC_UNKNOWN,
         })
     }
-    /// Add the GAP service to the attribute table.
-    pub fn build<M: RawMutex, const MAX: usize>(self, table: &mut AttributeTable<'a, M, MAX>) {
```

Comment:
There's probably a good reason this is removed and put directly in the macros now. This is a reminder for me to ask you what it is :)